lib/BackgroundJobs/ComputeStatistics.php

```diff
@@ -136,7 +136,7 @@ private function getNumericalEvaluatedStatistics($category, $key) {
 
 		$query = $this->connection->getQueryBuilder();
 		$result = $query
-			->select($query->createFunction('AVG(`value`) AS average, MAX(`value`) as max, MIN(`value`) as min'))
+			->select($query->createFunction('AVG(CAST(`value` AS SIGNED)) AS average, MAX(CAST(`value` AS SIGNED)) as max, MIN(CAST(`value` AS SIGNED)) as min'))
```

Comment:
This is not going to be executed on anything other then MySQL, therefor it's fine, but I don't think this works anywhere else because of
```
CAST(`value` AS SIGNED)
```
which is MySQL specific.
```php
->selectAlias($query->createFunction('AVG(CAST(`value` AS SIGNED))'), 'average')
```
would also be cleaner and removes troubles with the `AS` at least (as a general notice)